### DISCUSSION TITLE
Need two driver instance in appium webdriver io

### DISCUSSION THREAD

**Discussion by sivalive123** [webdriverio/webdriverio#11569] [Community]
In appium java client, i used to create two drivers for two device, and run my test. Device 1 and Device 2 both will have 2 diffrent logins. So i use 2 drivers to interact with in a single test. Is it possible in Webdriver io to create two instance for appium ?

**Comment by christian-bromann** [comment]
If you need these devices to interact with each other, I recommend to look into [multiremote](https://webdriver.io/docs/multiremote). If you just run the same tests but need different login credentials for all tests I recommend use the [shared store service](https://webdriver.io/docs/shared-store-service) and pull credentials via [resource pools](https://webdriver.io/docs/shared-store-service#resource-pools).
